src/js/utils/ui.js

```diff
@@ -201,6 +218,7 @@ function initInteractionListeners(ui) {
         // Always add this, in case we don't properly identify the device as mobile
         addEventListener(ui, initGroup, 'touchstart', interactStartHandler, listenerOptions);
     }
+    addEventListener(ui, initGroup, 'click', interactStartHandler);
```

Comment:
This additional listener causes `interactStartHandler` to fire on all UI instances on "click" and can result in these properties being modified between the expected mouse/pointer/touch handlers.
```
        ui.dragged = false;
        ui.lastStart = now();
        ui.startX = pageX;
        ui.startY = pageY;
```

When we want to actually listen for click, `initInteractionListeners` will be called from here:

```
click(ui) {
  initInteractionListeners(UI); 
```

So in that case, why don't we only listen for click? (To avoid the additional `interactStartHandler` for click events when we're really just trying to listen for drag events)